inc/saiport.h

```diff
@@ -2529,7 +2529,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
     /** End of custom range base */
-    SAI_PORT_ATTR_CUSTOM_RANGE_END
+    SAI_PORT_ATTR_CUSTOM_RANGE_END,
+
+    /**
+    * @brief Fabric port reachability for all remote reachable switches
+    *
+    * @type sai_u16_list_t
+    * @flags READ_ONLY
+    */
+    SAI_PORT_ATTR_FABRIC_REACHABILITY_VECTOR
```

Comment:
dont put attributes after SAI_PORT_ATTR_CUSTOM_RANGE_END,

Reply:
HI kcidnik,
I tried putting it in middle , saimetadata checks were failing by saying enum shift , do i need to makes some changes to handle that ?